contrib/cirrus/install_packaged_dependencies.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+set -e
+
+source $(dirname $0)/lib.sh
+
+req_env_var "
+    GOPATH $GOPATH
+    GOSRC $GOSRC
+    DEPENDENCY_VERSIONS_FILEPATH $DEPENDENCY_VERSIONS_FILEPATH
+    SUDOAPTGET $SUDOAPTGET
+    BIGTO $BIGTO
+    LILTO $LILTO
+    OS_RELEASE_ID $OS_RELEASE_ID
+    PACKGE_DEP $PACKAGE_DEPS
+"
+
+if [[ "$PACKAGE_DEPS" != "true" ]]
+then
+    echo "ERROR: Must install one of either package or source based dependencies"
+    exit 19
+fi
+
+echo "Installing/updating all packaged dependencies for podman"
+PACKAGES="
+    atomic-registries
+    buildah
+    container-selinux
+    containernetworking-plugins
+    containernetworking-plugins-devel
+    criu
+    libseccomp
+    podman
+    runc
+    skopeo-containers
+"
+VERSION_CMD="rpm -q --qf %{V}"
+case "$(os_release_id)" in
+    ubuntu)
+        PACKAGES="
+            buildah
+            containernetworking-plugins
+            cri-o-runc
+            libseccomp-dev
+            libseccomp2
+            podman
+            skopeo
+        "
+        INSTALL_CMD="$BIGTO ooe.sh $SUDOAPTGET install"
+        VERSION_CMD="dpkg-query --showformat '\${Version}' --show"
+        # Make sure we're using the latest and greatest
+        $LILTO $SUDOAPTGET update
+        $LILTO $SUDOAPTGET upgrade
+        ;;
+    fedora)
+        INSTALL_CMD='ooe.sh sudo dnf -y install'
```

Comment:
I guess here we'll include a copr enable lsm5/runc and add a line to disable runc updates from official fedora repos until we find out if it's all envs or just cirrus

Reply:
Try adding this:

    dnf install https://lsm5.fedorapeople.org/runc-1.0.0-60.dev.git578fe65.fc29.x86_64.rpm

(not adding the copr and config-manager steps atm)

Reply:
and for f28 there's

    dnf install https://lsm5.fedorapeople.org/runc-1.0.0-60.dev.git578fe65.fc28.x86_64.rpm

this is only until copr gets unblocked.

Reply:
maybe 

    dnf install dnf install https://lsm5.fedorapeople.org/runc-1.0.0-60.dev.git578fe65.fc${Version}.x86_64.rpm

is generic enough assuming ${Version} evaluates to 28 or 29

Reply:
now that copr is available:

    dnf -y copr enable lsm5/runc
    dnf -y install runc
    # exclude runc from being updated
    echo "exclude=runc" >> /etc/dnf/dnf.conf